include/mbedtls/config.h

```diff
@@ -2025,8 +2084,16 @@
  *          library/pem.c
  *          library/ssl_tls.c
  *
- * This module is required for SSL/TLS and X.509.
- * PEM_PARSE uses MD5 for decrypting encrypted keys.
+ * This module is required for SSL/TLS up to version 1.1, and for TLS 1.2
```

Comment:
I think we could emphasise the difference a little bit more, by saying "and can be used in TLS 1.2 ..."

Reply:
Done.